### PR TITLE
Updates docs for replacements api

### LINKED ISSUE #573: Replacements API documentation missing
Thanks for submitting a bug report, please fill in the following details.

**Description**
There is a link to the Replacements docs that appears to be broken

**Reproduction Steps**
1. Go to https://changie.dev/integrations/nodejs/
1. Click on the link to https://changie.dev/config/replacements/
1. See the 404 page

**What happened**
404 page with no docs on the replacement api

**Expected behavior**
I would expect to see some documentation of the code in `core/replacement.go`.

### PR DESCRIPTION
**Pull request by katerberg**
Closes #573 

**Check the following**
- [x] Keep 100% code coverage
- [x] Be properly formatted
- [x] Documentation changes are included
- [x] Include a change file if expected

**Additional context**

Demo:

Screenshot 2023-11-07 at 1 50 33 PM

### CODE REVIEW

File: .changes/unreleased/Fixed-20231107-140052.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+kind: Fixed
```
Comment (miniscruff) [inline .changes/unreleased/Fixed-20231107-140052.yaml:1]: I don't think a change fragment is required, there is a special action I can run to update the website without a release.

**Review by miniscruff — CHANGES_REQUESTED** [review]

File: .changes/unreleased/Fixed-20231107-140052.yaml (hunk 1 above)
Comment (katerberg) [reply .changes/unreleased/Fixed-20231107-140052.yaml]: Removed.

**Review by miniscruff — CHANGES_REQUESTED** [review]
Will merge and deploy after work

**Review by miniscruff — APPROVED** [review]
Meant approve...

**Pull request merged**